### ISSUE TITLE
Multi-target regression - one model or multiple submodels

### ISSUE THREAD

**Issue by krzjoa** [tensorflow/decision-forests#173]
Hi, 

I have a question regarding the multi-target regression tasks. I know how to do that with TFDF (passing a list of `MultiTaskItem(label=target_column, task=tfdf.keras.Task.REGRESSION)` objects), but I'm curious, how it's implemented.

Does YDF create a separate submodel for each target variable or use one model to predict all the target variables simultaneously?

**Comment by rstz** [comment]
Hi,

This is a part of the library that's under active development, so the answer might change soon. The last release of TF-DF still has independent models for each objective. However, a [recent commit](https://github.com/google/yggdrasil-decision-forests/commit/456420ba985ad8855e4542afb19c6dfa30c4c342) to YDF allowed Multi-task learning where the output of models trained as secondary are used as input for the "primary" model. This (and possibly more) changes to Multi-task models will land withe next TF-DF version.
